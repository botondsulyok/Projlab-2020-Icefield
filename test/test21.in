field spawn 3 false 0
entity exp Explorer spawn
ability spawn exp
stats spawn
stats exp