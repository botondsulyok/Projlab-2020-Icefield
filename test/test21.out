The iceunit can hold 3 objects!
id: spawn
type: unstable
building: plains0
snow: 0
pickable: null
standers: exp
pullers: exp
masters: exp
neighbours: null
capacity: 3
broken: false

id: exp
type: explorer
iceUnit: spawn
movebehaviour: walk0
bodyheat: 4
stamina: 3
digger: barehand0
waterProtecter: lung0
pullerDevice: basicpull0
tent: null
components: null

